Assets/Scripts/UI/PickupPup/MainMenu/MainMenu.cs

```diff
@@ -6,15 +6,20 @@
 
 using UnityEngine;
 using UnityEngine.UI;
+using UnityEngine.Advertisements;
 
 using k = PPGlobal;
+using System.Collections;
 
 public class MainMenu : PPUIElement
 {
     const int STANDARD_DROPDOWN = k.STANDARD_DROPDOWN;
     const int ALT_SINGLE_DROPDOWN = k.ALT_SINGLE_DROPDOWN;
 
-	[Header("External References")]
+    const string ANDROID_ID = "1347448";
```

Comment:
Is there a reason these hardcoded and not randomly generated GUIDs or some other instance: https://msdn.microsoft.com/en-us/library/system.guid.newguid(v=vs.110).aspx?